package.json

```diff
@@ -42,8 +42,8 @@
 		"postarchive": "unzip woocommerce-subscriptions-core.zip -d woocommerce-subscriptions-core && rm woocommerce-subscriptions-core.zip && zip -r woocommerce-subscriptions-core.zip woocommerce-subscriptions-core && rm -rf woocommerce-subscriptions-core"
 	},
 	"engines": {
-		"node": ">=0.8.0",
-		"npm": ">=1.1.0"
+		"node": "^16.17.1",
+		"npm": "^8.15.0"
```

Comment:
I thought this would limit usage/install to these versions, but it doesn't. TIL.